.github/workflows/deployment-tests.yml

```diff
@@ -0,0 +1,65 @@
+name: Deployment Tests
+
+on:
+  push:
+  pull_request:
+  schedule:
+    - cron: '0 0 * * *'
+
+jobs:
+  docker-builds:
+    if: github.event.schedule
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        os_container: ['centos7', 'centos8', 'ubuntu20_04', 'ubuntu18_04']
+    steps:
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v1
+      - name: Login to DockerHub
+        uses: docker/login-action@v1
+        with:
+          username: ${{ secrets.DOCKERHUB_USERNAME }}
+          password: ${{ secrets.DOCKERHUB_TOKEN }}
+      - name: Build and push
+        id: docker_build
+        uses: docker/build-push-action@v2
+        with:
+          push: true
+          file: containers/cobald-tardis-deployment-test-env/Dockerfile.${{ matrix.os_container }}
+          tags: matterminers/cobald-tardis-deployment-test-env:${{ matrix.os_container }}
+  deployment-linux:
+    needs: docker-builds
+    if: always() && (needs.docker-builds.result == 'success' || needs.docker-builds.result == 'skipped')
+      # Needs to be commented when testing locally with act, since it is not yet supported.
+      # https://github.com/nektos/act/pull/629
```

Comment:
Does this comment still apply? The linked issue suggests it is about ``needs``, which suggests it's about the preceding line. Perhaps moving it up a bit or marking the relevant block with ``# needs comment start`` ``# needs comment end`` fences could work?

Reply:
Yes, the problem here is related to `needs.docker-builds.result`, which is not support yet by `act`.

Reply:
Do you want me to move one line up?

Reply:
Nah, I guess it's fine. This will only be relevant for someone actually using `act` and the information should be enough for them.